autoload/which_key/view.vim

```diff
@@ -26,17 +26,23 @@ function! s:calc_layout(mappings) abort " {{{
   let smap = filter(copy(a:mappings), 'v:key !=# "name" && !(type(v:val) == s:TYPE.list && v:val[1] == "which_key_ignore")')
   let layout.n_items = len(smap)
   let displaynames = which_key#view#get_displaynames()
-  let length = values(map(smap,
-        \ 'strdisplaywidth(get(displaynames, toupper(v:key), v:key).'.
-        \ '(type(v:val) == s:TYPE.dict ? get(v:val, "name", "") : v:val[1]))'))
 
-  let maxlength = max(length) + g:which_key_hspace
+  let prefix_length = values(map(copy(smap),
+        \ 'strdisplaywidth(get(displaynames, toupper(v:key), v:key))'))
+  let suffix_length = values(map(smap, 'strdisplaywidth('.
```

Comment:
You forgot to delete this line :(.

Reply:
oops! tried to commit the suggestion directly from github. should be fixed now